datashader/transfer_functions.py

```diff
@@ -70,26 +70,38 @@ def _normalize_interpolate_how(how):
     raise ValueError("Unknown interpolation method: {0}".format(how))
 
 
-def interpolate(agg, low="lightblue", high="darkblue", how='cbrt'):
+def interpolate(agg, low=None, high=None, how='cbrt', cmap=None):
     """Convert a 2D DataArray to an image.
 
+    Data is converted to an image either by interpolating between a `low` and
+    `high` color [default], or by a specified colormap.
+
     Parameters
     ----------
     agg : DataArray
     low, high : color name or tuple, optional
         The color for the low and high ends of the scale. Can be specified
         either by name, hexcode, or as a tuple of ``(red, green, blue)``
-        values.
+        values. Defaults are "lightblue" and "darkblue", respectively.
```

Comment:
Should we deprecate low, high, delete it from all our examples, and remove it in the following release, since `cmap=[low,high]` is about as concise and is more powerful?